app/data/2_0001.json

```diff
@@ -1,106 +1,101 @@
-                                            "q_code": "54",
-                                            "type": "PositiveInteger"
-                                        },
-                                        {
-                                            "id": "number-of-employees-total",
-                                            "label": "Total number of employees",
-                                            "mandatory": true,
-                                            "q_code": "50",
-                                            "type": "PositiveInteger"
-                                        }
-                                    ],
-                                    "description": "An employee is anyone aged 16 years or over that your organisation directly pays from its payroll(s), in return for carrying out a full-time or part-time job or being on a training scheme.",
-                                    "guidance": [
-                                        {
-                                            "list": [
-                                                "all workers paid directly from this business’s payroll(s)",
-                                                "those temporarily absent but still being paid, for example on maternity leave"
-                                            ],
-                                            "title": "Include:"
-                                        },
-                                        {
-                                            "list": [
-                                                "agency workers paid directly from the agency payroll",
-                                                "voluntary workers",
-                                                "former employees only receiving a pension",
-                                                "self-employed workers",
-                                                "working owners who are not paid via PAYE"
-                                            ],
-                                            "title": "Exclude:"
-                                        }
-                                    ],
-                                    "id": "number-of-employees-question",
-                                    "title": "On {{exercise.start_date|format_date}} what was the number of employees for {{respondent.trad_as_or_ru_name}}?",
-                                    "type": "General"
-                                }
-                            ],
-                            "title": "Quarterly Business Survey"
-                        }
-                    ],
-                    "type": "Questionnaire"
-                },
-                {
-                    "type": "Summary",
-                    "id": "summary"
-                }
-            ],
-            "id": "qbs",
-            "title": "Quarterly Business Survey"
-        }
-    ],
+    "groups": [{
```

Comment:
As discussed the run_lint.sh script is ran in `run_tests.sh` but not in `run.sh`